presto-main/src/main/java/io/prestosql/server/PrestoServer.java

```diff
@@ -156,10 +154,7 @@ private static void updateConnectorIds(Announcer announcer, CatalogManager metad
         // get existing announcement
         ServiceAnnouncement announcement = getPrestoAnnouncement(announcer.getServiceAnnouncements());
 
-        // get existing connectorIds
-        String property = nullToEmpty(announcement.getProperties().get("connectorIds"));
-        List<String> values = Splitter.on(',').trimResults().omitEmptyStrings().splitToList(property);
-        Set<String> connectorIds = new LinkedHashSet<>(values);
+        Set<String> connectorIds = new LinkedHashSet<>();
 
         // automatically build connectorIds if not configured
         if (connectorIds.isEmpty()) {
```

Comment:
`connectorIds.isEmpty()` is always `true`. Please try to remove `connectorIds` variable and optimize the if block.